src/main.cpp

```diff
@@ -120,11 +120,11 @@ int main(int argc, char** argv)
     // Get correct case for Windows filenames (fixes #7139)
     for (const auto& file : parser.positionalArguments()) {
         const auto fileInfo = QFileInfo(file);
-        WIN32_FIND_DATA findFileData;
+        WIN32_FIND_DATAW findFileData;
         HANDLE hFind;
-        hFind = FindFirstFile(fileInfo.absoluteFilePath().toUtf8(), &findFileData);
+        hFind = FindFirstFileW(fileInfo.absoluteFilePath().toStdWString().c_str(), &findFileData);
```

Comment:
Why are you not using `String::toWCharArray()` here?

Reply:
Actually, doesn't matter. toWCharArray() would also require you to allocate a wchar_t* first, so this is fine, I suppose.

Reply:
Wouldn't `toUtf8().data()` do the trick?

Reply:
I am not sure this is more elegant, im getting
`argument of type "const ushort *" is incompatible with parameter of type "LPCWSTR"`
i would still have to convert to wchar_t...

Reply:
Cast, not convert.

Reply:
i could replace it with the following:
```c++
        const wchar_t* absolutePathWchar = (wchar_t*)fileInfo.absoluteFilePath().utf16();
        hFind = FindFirstFileW(absolutePathWchar, &findFileData);
```
not sure if this is better, tested this to work, couldnt do it without casting to wchar_t

Reply:
Yah that's the way but use modern casting

Reply:
Made changes at commit 43c0080a0a6cf5e41fcb6635bd4b330304b54c9c